### ISSUE TITLE
Affected_Rows for PostgreSQL fails with PHP 8.1+

### ISSUE THREAD

**Issue by michael-dev** [ADOdb/ADOdb#953]
In adodb-postgres64.inc.php:
 ```
function _affectedrows()
        {
                if (!is_resource($this->_resultid) || get_resource_type($this->_resultid) !== 'pgsql result') return false;
                return pg_affected_rows($this->_resultid);
        }
```
The if-condition evaluates to True due to $this->_resultid being object(PgSql\Result) and thus not !is_resource.
This results in phppgadmin erroring (and rolling back) on updating a row in a table.

**Comment by michael-dev** [comment]
The condition can be proved incorrect as pg_affected_rows returns a valid result to phppgadmin (int(1)). Maybe just remove it?

**Comment by dregad** [comment]
This issue has been fixed since ADOdb 5.22.2. Please see #833 (b8661ea2678995ec7a0efeb05e1dadcc957194fd)

In the future, please check whether a bug has already been fixed, before reporting a new issue for it. And when you do, kindly use our [bug report template](https://github.com/ADOdb/ADOdb/issues/new?assignees=&labels=triage&template=bug_report.md&title=), so we have all relevant information to analyze the problem. Thanks for your understanding
